config/config.go

```diff
@@ -24,6 +37,7 @@ func ParseConfig(filename string) (*Config, error) {
 		return nil, e
 	}
 	var config Config
+	SetDefaultConfig(&config)
```

Comment:
Collapse this line and the above line into a `NewConfig` or `NewDefaultConfig` function that sets the defaults. Convention in Go is that `NewX` functions handle all initialization. So instead of relying on me remembering to call `SetDefaultConfig` we can just make this more consistent with how Go types are usually initialized.

Reply:
addressed in https://github.com/jakevoytko/crbot/pull/60/commits/08ac2a782c80c3346c38af8e2cfee1daf0761df7